src/lib.rs

```diff
@@ -577,7 +675,7 @@ impl Subscription {
             for m in r.iter() {
                 if let Err(e) = handler(m) {
                     // TODO(dlc) - Capture for last error?
-                    println!("Error in callback! {:?}", e);
+                    eprintln!("Error in callback! {:?}", e);
```

Comment:
These calls to `eprintln` (same as println but writes to stderr) should probably be replaced by using the `log` crate and using the `error!` macro, or going further and using structured logging.

Reply:
On my todo list to capture and allow access. Two version, one for async server errors returned and one per subscription that has a handler for the handler returning an error.